components/Footer/index.js

```diff
@@ -0,0 +1,357 @@
+      marginBottom: 0
+    },
+    '& li': {
+      ...fontStyles.sansSerifRegular16,
+      marginTop: 8,
+      marginRight: 22,
+      [mediaQueries.mUp]: {
+        marginRight: 0
+      }
+    },
+    '& li:first-child': {
+      ...fontStyles.sansSerifRegular12,
+      marginTop: 0,
+      width: '100%',
+      [mediaQueries.mUp]: {
+        width: 'inherit'
+      }
+    }
+  })
+}
+
+const Footer = ({ t, me, signOut, inNativeIOSApp, isOnMarketingPage }) => {
+  const [colorScheme] = useColorContext()
+  const navLinkStyle = useMemo(
+    () =>
+      css({
+        ...fontStyles.sansSerifRegular16,
+        color: colorScheme.getCSSColor('text'),
+        textDecoration: 'none',
+        '@media (hover)': {
+          ':hover': {
+            textDecoration: 'underline'
+          }
+        }
+      }),
+    [colorScheme]
+  )
+
+  const FooterNavLink = props => <Link prefetch={false} {...props} />
+
+  const router = useRouter()
+
+  return (
+    <div {...styles.bg}>
+      <div {...styles.content}>
+        <div {...styles.topRow}>
+          <FooterNavLink href='/'>
+            <a>
+              {isOnMarketingPage ? (
+                <div style={{ width: 23, display: 'inline-block' }}>
+                  <BrandMark {...colorScheme.set('fill', 'text')} />
+                </div>
+              ) : (
+                <Logo {...colorScheme.set('fill', 'text')} height={20} />
+              )}
+              <span {...colorScheme.set('color', 'textSoft')} {...styles.since}>
+                {t('footer/since')}
+              </span>
+            </a>
+          </FooterNavLink>
+
+          <SocialLinks />
+        </div>
+        <hr {...styles.hr} {...colorScheme.set('borderColor', 'divider')} />
+        <div {...styles.middleRow}>
+          <div {...styles.nav}>
+            <ul {...styles.navList}>
+              <li {...colorScheme.set('color', 'disabled')}>
+                {me ? 'Meine Republik' : 'Mitglied werden'}
+              </li>
+              {!!me && (
+                <>
+                  <li>
+                    <FooterNavLink href='/konto'>
+                      <a {...navLinkStyle}>
+                        {t(me ? 'footer/me/signedIn' : 'footer/me/signIn')}
+                      </a>
+                    </FooterNavLink>
+                  </li>
+                  <li>
+                    <FooterNavLink href={`/~${me.username || me.id}`}>
+                      <a {...navLinkStyle}>{t('footer/me/profile')}</a>
+                    </FooterNavLink>
+                  </li>
+                </>
+              )}
+              {!inNativeIOSApp && (
+                <li>
+                  <FooterNavLink
+                    href={{
+                      pathname: '/angebote',
+                      query: { group: me ? 'GIVE' : undefined }
+                    }}
+                  >
+                    <a {...navLinkStyle}>
+                      {t(me ? 'footer/me/give' : 'footer/offers')}
```

Comment:
we should check for `hasActiveMembership` from `withMemberStatus`. One might be logged in but not yet a active member

Reply:
potentially we could also show a list of options directly like on the imprint
<img width="481" alt="Screenshot 2021-05-27 at 15 51 38" src="https://user-images.githubusercontent.com/410211/119838195-7e515a80-bf03-11eb-8aa0-a0a449464058.png">

And since we habe space when not logged in we could show the gift/give link always. Not logged in can also buy for someone else.

Reply:
didn't do the options as it's mostly a layout PR.